### ISSUE TITLE
Breaking change from 0.3.1 to 0.3.2 - can't serialize collection created via pipeline/write-output

### ISSUE THREAD

**Issue by gaelcolas** [cloudbase/powershell-yaml#27]
Hi @gabriel-samfira,

I love this module, although I found a breaking change since 0.3.2 which can't serialise the collection returned from the pipeline (unless it's streamed to Convertto-Yaml).

Example:
This works in `0.3.1` but not in `0.3.2`
```PowerShell
Convertto-yaml -Data (Write-output 'a','b','c')
```

This works on both:
```PowerShell
Convertto-yaml -Data (Write-output -NoEnumerate 'a','b','c')
(Write-output -NoEnumerate 'a','b','c') | Convertto-yaml
```

Any chance for a fix?

**Comment by gabriel-samfira** [comment]
Quick test. Can you try:

```powershell
Convertto-yaml -Data @('a','b','c')
```

and

```powershell
 @('a','b','c') | ConvertTo-Yaml
```

**Comment by gaelcolas** [comment]
Both works. It's only when going through pipeline. :/

**Comment by gabriel-samfira** [comment]
Looks like doing

```powershell
(Write-Output "a", "b", "c")
```
as oposed to

```powershell
@("a", "b", "c")
```

produces a different powershell object. Using a plain array object, works. Using ```Write-Output``` to generate the array blows up. Investigating now.

**Comment by gabriel-samfira** [comment]
just pushed a potential fix. Can you clone master and give it a shot?

**Comment by gaelcolas** [comment]
Not in front of a PC right now, will try later tonight. Thanks!

**Comment by gabriel-samfira** [comment]
No worries. If it fixes the issue for you, please click on close issue :).

Cheers!

**Comment by gaelcolas** [comment]
Yep that fixed it! 

Thanks.
closing the issue, hope you release soon :)

**Comment by gabriel-samfira** [comment]
Released as version 0.3.3. Please test it out and let me know if all is well.
